rep-0128.rst

```diff
@@ -183,17 +183,17 @@ Let us say we have several catkin workspaces in our home folder (~/).
 ::
 
  ~/workspace1/
-     /src
-     /build
-     /devel
+      src/
+      build/
+      devel/
  ~/workspace2/
-     /src
-     /build
-     /devel
+      src/
+      build/
+      devel/
  ~/workspace3/
-     /src
-     /build
-     /devel
+      src/
+      build/
+      devel/
```

Comment:
Please don't use 5 spaces for indentation but 4 as before the patch.

Reply:
Thank you for the review. Fixed.